src/hydrate/values/readTitleFromReadme.test.ts

```diff
@@ -0,0 +1,33 @@
+import { describe, expect, it, vi } from "vitest";
+
+import { readTitleFromReadme } from "./readTitleFromReadme.js";
+
+const mockReadFileSafe = vi.fn();
+
+vi.mock("../readFileSafe.ts", () => ({
```

Comment:
[Typo] Interesting that this works regardless...

```suggestion
vi.mock("../readFileSafe.js", () => ({
```